R/capacity/disasters-emergencies/england/build-index.R

```diff
@@ -0,0 +1,39 @@
+# ---- Load libraries and Functions ----
+library(tidyverse)
+library(demographr)
+
+source("R/utils.R")
+
+# ---- Load indicators ----
+indicators <-
+  load_indicators(
+    path = "data/capacity/disasters-emergencies/england",
+    key = "lad_code"
+  )
+
+# Check entry for all 2021 LAD codes
+lads_21 <- lookup_lad_counties_21 |>
+  filter(str_detect(lad_code, "^E")) |>
+  distinct(lad_code) 
+
+lads_21 |>
+  anti_join(indicators, by = "lad_code")
+
+indicators |>
+  anti_join(lads_21, by = "lad_code")
+
+indicators |>
+  dplyr::filter(if_any(everything(), ~is.na(.x))) 
+# City of London & Isle of Scilly are both missing civic assests & engagement
+# Isle of Scilly also missing station response time
+# Assumption: drop these LADs due to too many missing variables. 
+
+indicators_complete <- indicators |>
+  drop_na()
+
+# ---- Build Index ----
```

Comment:
Do `la-spending-power` not need inverting/rescaling (*-1) before computing domain scores as higher score = higher capacity, which is in the opposite direction to the other variables?

Reply:
Oops yes of course, thanks for flagging this. Fixed this and also swapping direction at end of script from [here](https://britishredcrosssociety.github.io/resilience-index-book/technical.html#stage-3-realign-files-ranks-deciles). Changes made in commit https://github.com/britishredcrosssociety/resilience-index/commit/17c4b5fa81e7e09e5d64d7d9191af6166c2c2806.